src/IBM.WatsonDeveloperCloud.Discovery.v1/Model/QueryResponse.cs

```diff
@@ -34,7 +34,7 @@ public class QueryResponse : BaseModel
         /// Gets or Sets Results
         /// </summary>
         [JsonProperty("results", NullValueHandling = NullValueHandling.Ignore)]
-        public List<QueryResult> Results { get; set; }
+        public List<dynamic> Results { get; set; }
```

Comment:
these are handwritten changes in the .NET SDK - I'm working on transitioning to something closer to the Java SDK next major release. This round I was searching for anything that I missed that had additionalProperties. You're right though - this one doesn't. Let me take another look at these to make sure I didn't miss any others.

Reply:
Hmm, `QueryResult` has additional properties - should this be generated as dynamic?
https://github.ibm.com/Watson/developer-cloud--api-definitions/blob/a998e9540b7bd3a5c300cb2f940ba8efd3233253/apis-public/discovery-v1.json#L9320-L9347

And does that mean `QueryNoticesResult` should be generated as dynamic as well?
https://github.ibm.com/Watson/developer-cloud--api-definitions/blob/a998e9540b7bd3a5c300cb2f940ba8efd3233253/apis-public/discovery-v1.json#L9440-L9443

I think I may be confusing myself here

Reply:
Ah, java `QueryResult` is generated as dynamic

https://github.com/watson-developer-cloud/java-sdk/blob/master/discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/QueryResult.java

I think the confusion is that while in Java the class extends map, in .NET we declare the model that would extend map as `dynamic` - one level up. This is something I really want to change in the next major release but for now I think this instance is right.

Reply:
That makes sense now. But won't this change the way the user interacts with each element of the `Results` property?

Reply:
In this case it's a response. When it was typed as `QueryResult` before we would lose any additional data that got sent in the response. Now the type would be evaluated at runtime. 

Their current code would still work though. If they were trying to evaluate a `Score` for example it wouldn't give an error unless no `Score` is found at runtime

```
// still works when the result is dynamic type
var resultScore = result.Results[0].Score
```

The only issue I see is that if they are trying to use intellisense to see what properties are available in the result it wouldn't show anything since the object is now dynamic. 

The other issue is when creating a dynamic object we don't see what is in the object using intellisense either

```
dynamic result = new QueryResult();
// Score will not show up in autocomplete - but since its dynamic you can put whatever you want in there.
result.Score = 0.1;
```

Anyway, if you think we shouldn't go through with changing these to dynamic right now I'm fine with that decision as well. We are pretty close to the major release either way!

Reply:
No I think that's good! Just wanted to make sure no code would break really. And like you said, it's close to a major release anyway.